### PR TITLE
Add contributing file and PR template

### PR DESCRIPTION
**Pull request by dadah89**
It'd be nice to have a reminder in the PR template asking to update the Changelog. Opened this PR for discussion, not sure this is the best way to go about it, let me know if you have any suggestions.

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+#### Development
+
+When opening a pull request, add your changes to the "[Unreleased]" section of the [changelog](https://github.com/zendesk/active_record_shards/blob/master/Changelog.md).
+
+#### Release
+
+After merging your changes into master, cut a tag and push it immediately:
+
+1.  Update the version by `bundle exec rake bump:patch`, `bundle exec rake bump:minor` or `bundle exec rake bump:major`.
```
Comment (bquorning) [inline Contributing.md:12]: Do people really use `rake bump`? I always just manually change the version file.

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (bquorning) [inline Contributing.md:1]:
```suggestion
```

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+### Committing a change
+
+#### Development
+
+When opening a pull request, add your changes to the "[Unreleased]" section of the [changelog](https://github.com/zendesk/active_record_shards/blob/master/Changelog.md).
+
+#### Release
+
+After merging your changes into master, cut a tag and push it immediately:
```
Comment (bquorning) [inline Contributing.md:10]: I would actually prefer that people open a pull request with the version bump.

File: Contributing.md (hunk 1 above)
Comment (mervync) [reply Contributing.md:12]: Should we remove `bump` from the gemspec then?

Comment (dadah89) [reply Contributing.md:11]: I use `bundle exec bump` :) But yeah, in most repositories (like here I assume) rake just changes the version file, so seems fine I think manually updating it as part of the release steps.

Comment (dadah89) [reply Contributing.md:11]: Just realized there is no version.rb file in the gem oO Updating manually still seems fine though

Comment (dadah89) [reply Contributing.md]: updated to tell people to just modify the gemspec file

File: Contributing.md (hunk 3 above)
Comment (dadah89) [reply Contributing.md:9]: updated the steps to the list above

**Pull request closed without merging**